ksql-parser/src/test/java/io/confluent/ksql/parser/tree/SpanExpressionTest.java

```diff
@@ -16,26 +16,24 @@
 
 package io.confluent.ksql.parser.tree;
 
-import static com.google.common.base.MoreObjects.toStringHelper;
+import org.junit.Test;
 
-public class NaturalJoin
-    extends JoinCriteria {
+import java.util.concurrent.TimeUnit;
 
-  @Override
-  public boolean equals(Object obj) {
-    if (this == obj) {
-      return true;
-    }
-    return (obj != null) && (getClass() == obj.getClass());
-  }
+import static org.testng.AssertJUnit.assertEquals;
+
+public class SpanExpressionTest {
 
-  @Override
-  public int hashCode() {
-    return 0;
+  @Test
+  public void shouldDisplayCorrectStringWithSingleSpan() {
```

Comment:
Wonder if we should create the `JoinWindows` in the ctor of `SpanExpression` - it would then throw an exception early on if the `JoinWindows` are invalid?

Reply:
Good suggestion. done.